### DISCUSSION TITLE
How can I prevent the user from going to the next slide?

### DISCUSSION THREAD

**Discussion by leonardoprogrammer** [AppIntro/AppIntro#1057] [Q&A] [answered]
Hello.

I added a fragment with Spinner to my slides and I need to prevent it from going to the next slide if the user doesn't select an item from Spinner.
I can't find a method that blocks the pass.
Can someone help me?

**Comment by cortinico** [comment, marked as answer]
You will have to use the SlidePolicy interface:
https://github.com/AppIntro/AppIntro#slide-policy

There is also an example in the sample app with a slider that has a checkmark that needs to be checked in order for the user to proceed.
